src/shared/ui/kit/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./mui";
```

Comment:
redundant export

Reply:
had a conversation about usage of this export with you, it is fine